scripts/Dockerfile.branch

```diff
@@ -0,0 +1,15 @@
+ARG DOCKER_IMAGE_TAG=11.3.1-devel-ubuntu20.04
```

Comment:
Im pretty confused...there seem to be some files and changes from infra team that are not in master but are in our YoloNas branch...

Reply:
Should I move these changes to another PR or is it ok to leave it here?

Reply:
Would appreciate if you do, but dont have to.